### ISSUE TITLE
Explanatory not talking to ICEES

### ISSUE THREAD

**Issue by vgardner-renci** [NCATSTranslator/minihackathons#70]
https://arax.ncats.io/?source=ARS&id=0cdc5dab-49ba-4a3d-9688-8f36f95b964d

**Comment by Rosinaweber** [comment]
Yes, not at the current version. It will take some time until the next version is deployed because we are making substantial changes.

**Comment by vgardner-renci** [comment]
Sb released by September relay.
